### PR TITLE
fix description file

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by Mashimiao** [comment]
There are multiple errors, please modify based on the code of oci-runtime-tool.

**Comment by q384566678** [comment]
@Mashimiao  updated.

File: man/oci-runtime-tool-validate.1.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Validate an OCI bundle
 **--help**
   Print usage statement
 
-**--path=PATH
-  Path to bundle
+**--path**=PATH
+  Path to bundle. The default is *.*.
```
Comment (Mashimiao) [inline man/oci-runtime-tool-validate.1.md]: I think change this to `The default is current working directory.` maybe better.

Comment (q384566678) [inline man/oci-runtime-tool-validate.1.md]: Yes, it will be much better

**Comment by Mashimiao** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**